File: Taxi Booking[21-March-2023]/Solution.java
class Solution {
    public static int minimumTime(int N, int cur, int[] pos, int[] time) {
        
        int ans=Integer.MAX_VALUE;
        
        for(int i=0;i<N;i++)
        {
            ans=Math.min(ans,Math.abs(cur-pos[i])*time[i]);
        }
        
            return ans;
    }
}

File: Taxi Booking[21-March-2023]/solution.cpp
class Solution{
public:
    int minimumTime(int N,int cur,vector<int> &pos,vector<int> &time){
        int ans=1e9;
        for(int i=0;i<N;i++){
            int dist=abs(pos[i]-cur);
            ans=min(ans,dist*time[i]);
        }
        return ans;
    }
};